Utils/API/server/lib/dkb/api/__init__.py

```diff
@@ -10,7 +10,7 @@
 CONFIG_DIR = '%%CFG_DIR%%'
 
 
-__version__ = '0.2.dev20191203'
+__version__ = '0.2.dev20200129'
```

Comment:
@Evildoor, just out of curiosity: why not today, but 0129? :)

Reply:
Because 29 of January is the date of 82f4970, the last change (that is not update of the version). "Version = date of last real change" sounds better to me than "version = date of last real change + some time has passed before reviewer had an opportunity to review the changes and point out that version should be changed + some time has passed before developer could see the comment and make the change". Not that much better - after all, these "real" changes are also dependent on "when I have time to get to reviewing/fixing/discussing/etc." - but still. 

Also, in this way life would probably be simpler in a hypothetical case when someone looks through old commits (and wonders why there was a pause of N days between some activity and version update otherwise).

Reply:
> "Version = date of last real change" sounds better to me than "version = date of last real change + some time has passed before reviewer had an opportunity to review the changes and point out that version should be changed + some time has passed before developer could see the comment and make the change".

It makes sense, so why not.

My thoughts on this idea, although, were almost opposite: "version = something close to the moment when the changes are getting to the production". Why: if we have multiple changes done in parallel branches, it'd be better to have version in production saying "when the last change was applied", not "when the last change was suggested" -- since the one suggested later may for some reason be applied earlier, and the next applied change may set the version to an earlier date... which is ridiculous.

Although in such case there will be conflicts, right? And if we solve them setting version to the date of 'conflict solving' moment, it'll do the job.

Reply:
Aha, I see your point. Yes, I think that suggested procedure with different branches will do. And if it won't, then we would probably need to start thinking about more thoroughly planned version system.